File: 2-data-types/data-types.vb
Imports System
Module Solution
    Public Shared Sub Main()
        Dim i As Integer = 4
        Dim d As Decimal = 4.0
        Dim s As String = "HackerRank "

        Dim i2 As Integer = Console.ReadLine()
        Dim d2 As Decimal = Console.ReadLine()
        Dim s2 As String = Console.ReadLine()

        Console.WriteLine(i+i2)
        Console.WriteLine((d+d2).toString("F1"))
        Console.WriteLine(s+s2)
    End Sub
End Module

File: 2-data-types/data-types.ex
defmodule Solution do
  def hello(i2, d2, s2) do
    i = 4
    d = 4.0
    s = "HackerRank "
    i3 = i + i2
    d3 = d + d2
    s3 = "#{s}#{s2}"
    IO.puts (i3)
    IO.puts (d3)
    IO.puts (s3)
  end
end

# i2 = IO.gets("")
# d2 = IO.gets("")
# s2 = IO.gets("")
# Solution.hello(i2, d2, s2)
